extensions/wikia/Discussions/maintenance/dumpForumData.php

```diff
@@ -25,11 +25,12 @@ public function __construct() {
 	}
 
 	public function execute() {
-		if ( $this->hasOption( 'out' ) ) {
-			$this->fh = fopen( $this->getArg(), 'w' );
-		} else {
-			$this->fh = STDOUT;
+		$outputName = $this->hasOption( 'out' ) ? $this->getArg() : "php://stdout";
+		$this->fh = fopen( $outputName, 'w' );
+		if ( $this->fh === false ) {
+			$this->error( "Unable to open file " . $outputName, 1 );
```

Comment:
The `1` we're passing to the `error` method has the script die with a return value of 1. That return value is checked [here](https://github.com/Wikia/pandora/blob/master/service/discussion/src/main/java/com/wikia/discussionservice/tools/export/batchmigration/mediawiki/WikiaExporter.java#L28), which halts the migration process.